src/controllers/accounts/AccountModel.cpp

```diff
@@ -31,12 +31,12 @@ int AccountModel::beforeInsert(const std::shared_ptr<Account> &item,
 {
     if (this->categoryCount_[item->getCategory()]++ == 0)
     {
-        auto row = this->createRow();

```

Comment:
Something like `newRow` or leaving it as `row` is better. `created` doesn't carry much meaning.